Src/Support/GoogleApis/Apis/[Media]/Upload/ResumableUpload.cs

```diff
@@ -817,7 +915,7 @@ private HttpRequestMessage CreateInitializeRequest()
 
             // init parameters
             builder.AddParameter(RequestParameterType.Query, "key", Service.ApiKey);
-            builder.AddParameter(RequestParameterType.Query, "uploadType", "resumable");
+            builder.AddParameter(RequestParameterType.Query, UploadType, Resumable);

```

Comment:
The constants were already in place, just unused, which I felt was an oversight I should correct. Like I said, I was trying to err of the side of disturbing as little as possible, so introducing new constants seemed like it wasn't worth it. But I can if you'd like.